content/en/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -78,7 +78,7 @@ unless the Pod's grace period expires. For more details, see
 [Pod Lifecycle](/docs/concepts/workloads/pods/pod-lifecycle/).
 
 {{< note >}}
-Kubernetes only sends the preStop event when a Pod is *terminated*.
+Kubernetes only sends the preStop event when a Pod or a container in the Pod is *terminated*.
 This means that the preStop hook is not invoked when the Pod is *completed*. 
 This limitation is tracked in [issue #55087](https://github.com/kubernetes/kubernetes/issues/55807).
```

Comment:
BTW, this line also looks out of date.

Reply:
So can I delete L83?

Reply:
Is the explanation correct
PreStop hook is used to gracefully terminate the container(s) with some commands specified during the declaration of container in pod, deployment, etc. if it fails to execute or it gets hanged and it passes the graceTimeOutPeriod then there is a crashBackLoop this indicates the container has stopped with error(s)
like to see your views @sftim